frontend/appflowy_flutter/packages/flowy_infra_ui/example/macos/Podfile

```diff
@@ -1,4 +1,4 @@
-platform :osx, '10.11'
+platform :osx, '10.14'
```

Comment:
Is it necessary to update this?

Reply:
actually, it wasn't working before updating this. showed some warning.

i'll just test it again and if it works, will change it back to `10.11`